lightning/src/ln/channelmanager.rs

```diff
@@ -11987,6 +11987,12 @@ where
 	}
 
 	fn handle_open_channel_v2(&self, counterparty_node_id: PublicKey, msg: &msgs::OpenChannelV2) {
+		if !self.node_features().supports_dual_fund() {
```

Comment:
Looks like we use `init_features` to check for quiescence support. Though maybe we could check the `UserConfig` instead of constructing the features?

Reply:
I just match quiescence by checking `init_features` for now, as I imagine we'll eventually have it enabled by default.